backend/src/main/java/com/woowacourse/levellog/team/application/TeamService.java

```diff
@@ -31,20 +20,16 @@
 public class TeamService {
 
     private final TeamRepository teamRepository;
-    private final ParticipantRepository participantRepository;
     private final MemberRepository memberRepository;
     private final TimeStandard timeStandard;
 
     @Transactional
     public Long save(final TeamWriteDto request, final Long hostId) {
         final Member host = memberRepository.getMember(hostId);
-        final Team team = request.toEntity(host.getProfileUrl());
-        final Participants participants = createParticipants(team, hostId, request.getParticipantIds(),
-                request.getWatcherIds());
-        team.validParticipantNumber(participants.size());
+        final Team team = new Team(request.toTeamDetail(host.getProfileUrl()),
+                request.toParticipantsIngredient(hostId));
```

Comment:
```suggestion
        final Team team = request.toEntity(host.getProfileUrl());
```
dto에게 team 엔티티의 생성 책임을 위임하는건 어떨까요?